### ISSUE TITLE
issue with pawn.json for older versions of plugin

### ISSUE THREAD

**Issue by ADRFranklin** [pBlueG/SA-MP-MySQL#201]
So older versions of the release packages do not follow the same folder structure as the newer ones do, and because of this sampctl users are not able to download/extract resources and this leaves them with an empty directory.

a possible fix is to change the regex to make the first folder in the release package optional so that older versions can be extracted properly.

```
(mysql-.+\\/)?plugins\\/mysql.so
(mysql-.+\\/)?plugins\\/mysql.dll
```

This has not been tested but should be enough to fix this.

Issues found on sampctl:
https://github.com/Southclaws/sampctl/issues/258

**Comment by maddinat0r** [comment]
There's a lot of other things to consider:
 - there is no log-core library in R39
 - the R39 Linux package doesn't have any include files
 - there is a `libmysql.dll` instead of an `libmariadb.dll` in the R39 Windows package
 - there is no extra packaged static Linux plugin for R39, instead the static plugin build is bundled with the normal Linux package, so there are two plugin files, one being the normal `mysql.so`, and the static one, `mysql_static.so`

I don't know how sampctl would handle some of these things, like missing files for example. But I think there's a lot more to change to make the R39-* releases work.

**Comment by ADRFranklin** [comment]
Yeah, that is a lot of changes, and sampctl relies solely on your `pawn.json` for it to work. 

I'll talk with Southclaws about a possible update that could allow you to specify a way to handle changes like seen here.
